lib/find_non_repeated_letter.rb

```diff
@@ -9,6 +9,28 @@
 
 
 def find_first_non_repeated_letter(str)
-  #write your method here
-  return true
+  
+  array = str.chars
+  
+  final = array.length - 1
+  
+  answer = ""
+  
+  array.each_index do |i|
+    if i == 0 && array[i] != array[(i+1)]
```

Comment:
I'm a little confused why we wouldn't need a loop, if we need to find the *first instance* of a non-repeating letter. Even in thinking about shorter ways to solve this, potentially with a hash of character/count key value pairs similar to another exercise, we would still have to look through the array of characters.